ui/packages/catalog-ui-search/src/main/webapp/react-component/container/workspaces-templates-container/workspaces-templates-container.tsx

```diff
@@ -36,8 +38,16 @@ class WorkspacesTemplatesContainer extends React.Component<Props, State> {
     }
   }
   startAdhocSearch() {
-    this.prepForCreateNewWorkspace()
-    store.get('workspaces').createAdhocWorkspace(this.state.value)
+    try {
+      store.get('workspaces').createAdhocWorkspace(this.state.value)
+      this.prepForCreateNewWorkspace()
```

Comment:
❓ Does the order of these two lines not matter?

Reply:
It looks like `prepForCreateNewWorkspace` actually navigates to the new workspace. 'navigateToNewWorkspace` would make more sense in my opinion

Reply:
Yes, that was the reason. The prep function sets things up to immediately load the page for the new workspace; if we are not actually creating the workspace because of the parsing issues, the page just gets stuck on a loading/spinning wheel. I can probably rename the function

Reply:
This function is now named `navigateToNewWorkspace`